hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/hudi/streaming/HoodieStreamSource.scala

```diff
@@ -163,10 +178,7 @@ class HoodieStreamSource(
     startOffset match {
       case INIT_OFFSET => startOffset.commitTime
       case HoodieSourceOffset(commitTime) =>
-        val time = HoodieActiveTimeline.parseDateFromInstantTime(commitTime).getTime
```

Comment:
why this was removed? was there any bug?

Reply:
I think this is unnecessary, since we filter all instants in `IncrementalRelation` and `MergeOnReadIncrementalRelation` by methods `findInstantsInRangeByStateTransitionTs` and `findInstantsInRange`, these two methods filter instants after startTs and before or on endTs.

```java
/**
   * Create a new Timeline with instants after startTs and before or on endTs.
   */
  HoodieTimeline findInstantsInRange(String startTs, String endTs);

  /**`
   * Create a new Timeline with instants after startTs and before or on endTs
   * by state transition timestamp of actions.
   */
  HoodieTimeline findInstantsInRangeByStateTransitionTs(String startTs, String endTs);
```